### PR TITLE
Update cuckoo-filter.md

### PR DESCRIPTION
**Pull request by sav-norem**
update cuckoo md to include tabbed example shortcodes

### CODE REVIEW

**Comment by sav-norem** [comment]
@dwdougherty if you could check this out and let me know if it needs any fixes!

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Style changes and a question about the capacity used.

I did a test build and it looks fine in situ.

File: docs/docs/cuckoo-filter.md [hunk 1]
```diff
@@ -39,43 +39,20 @@ Note> In addition to these two cases, Cuckoo filters serve very well all the Blo
 
 ## Examples
 
-* Cuckoo: Adding new items to a filter
+> We'll ddemonstrate creating an empty cuckoo filter with an initial capacity (of 1000 items), adding items to it, checking if they exist, and deleting them. While the `CF.ADD` command will create a new filter if one does not exist, it likely will not be optimally sized for your use case. We recommend using `CF.RESERVE` to create a filter with the desired capacity.
```
[the remark this answers is not in the document]
Comment (sav-norem) [reply docs/docs/cuckoo-filter.md:42]: qq: is that quotation on the end missing an open quote or am I missing something else? also yes - that capacity is 1,000,000 so I'll correct that

[the remark this answers is not in the document]
Comment (dwdougherty) [reply docs/docs/cuckoo-filter.md:42]: Definitely a stray quotation mark. Sorry 'bout that.

**Review by dwdougherty — APPROVED** [review]
LGTM

**Comment by sav-norem** [comment]
merged into PR #670

**Pull request closed without merging**